packages/app/components/profile/profile-error.tsx

```diff
@@ -0,0 +1,139 @@
+import { Platform, Linking } from "react-native";
+
+import * as Clipboard from "expo-clipboard";
+
+import { Alert } from "@showtime-xyz/universal.alert";
+import { Button } from "@showtime-xyz/universal.button";
+import { useRouter } from "@showtime-xyz/universal.router";
+import { Text } from "@showtime-xyz/universal.text";
+
+import { View } from "design-system";
+
+import { Link, TextLink, LinkProps } from "../../navigation/link";
+import { EmptyPlaceholder } from "../empty-placeholder";
+
+const LinkComponent = (props: LinkProps) => {
+  if (Platform.OS === "web") {
+    return <Link {...props} />;
+  }
+  return <>{props.children}</>;
+};
+
+const Profile404 = () => {
+  return (
+    <View tw="items-center justify-center px-4 pt-8">
+      <EmptyPlaceholder
+        title="This user does not exist."
+        text={
+          <Text tw="text-center">
+            Try searching for another one or check the link.&nbsp;
+            <TextLink href={`/`} tw="text-indigo-500">
+              Go Home
+            </TextLink>
+          </Text>
+        }
+        hideLoginBtn
+      />
+    </View>
+  );
+};
+const Profile403 = () => {
+  const router = useRouter();
+  return (
+    <View tw="items-center justify-center px-4 pt-8">
+      <View tw="max-w-sm text-center">
+        <EmptyPlaceholder
+          title="Restricted"
+          text={
+            <Text tw="text-center">
+              This profile has been suspended for violating our&nbsp;
+              <TextLink
+                href="https://www.notion.so/showtime-xyz/Legal-Public-c407e36eb7cd414ca190245ca8621e68"
+                tw="text-indigo-500"
+                target="_blank"
+              >
+                Terms of Service
+              </TextLink>
+              . If you think that was a mistake, please let us know.
+            </Text>
+          }
+          hideLoginBtn
+        />
+      </View>
+      <View tw="mt-6 w-full max-w-xs flex-row justify-between px-4">
+        <LinkComponent href="/">
+          <Button
+            {...Platform.select({
+              web: {},
+              default: {
+                onPress: () => {
+                  router.replace("/");
+                },
+              },
+            })}
+          >
+            Take me home
+          </Button>
+        </LinkComponent>
+        <LinkComponent href="mailto:help@showtime.xyz">
+          <Button
+            {...Platform.select({
+              web: {},
+              default: {
+                onPress: async () => {
+                  const isCanOpen = await Linking.canOpenURL(
+                    "mailto:help@showtime.xyz"
+                  );
+                  await Clipboard.setStringAsync("help@showtime.xyz");
```

Comment:
need to remove this from here? as we also add the copy in the `else`

Reply:
yeah, my bad for the test on local, will update it.